aips/aip-116.md

```diff
@@ -91,7 +87,7 @@ The new standard introduces new interfaces that conform to [CAIP-122](https://ch
 ### Definitions
 
 - _binding_: The process of adding or modifying fields to fit requirements, sourcing fields from a trusted data source such as the browser window. This process will typically be done by the wallet.
-- _SIWA Message_: A plain text message that is constructed from the `AptosSignInInput` fields by the wallet. This message should follow the format specified in the [Message Construction](#message-construction) section.
```

Comment:
I would not capitalize "Message" here, since you're gonna use it in lowercase as "message" throughout the text. Same for others.